topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml

```
@@ -112,4 +106,38 @@ Resources:
       StorageCapacityMax: 50
       StorageCapacityMin: 25
       TapeCapacity: 0
+      UConn-OSG_CE:
```

```suggestion
  UConn-OSG_CE:
```

Looks like this is starting a new resource, so I'm guessing the indent should be reset..?

---

Please fix it, I cannot see it in full context here but it looks like there are too many spaces in front of this resource name. -rtj